pkgs/servers/hasura/default.nix

```diff
@@ -4,13 +4,12 @@ with haskell.lib;
 
 let
   # version in cabal file is invalid
-  version = "1.2.1";
+  version = "1.3.1-beta";
 
-  pkgs = haskell.packages.ghc865.override {
+  pkgs = haskell.packages.ghc8101.override {
```

Comment:
Out of curiosity. Is it important that you build this with 8.10 and not with 8.8 which is the current default in nixpkgs?

That's no deal breaker, but It would have the advantage of much more dependency reuse.

Reply:
GHC 8.8 will stay the default compiler for the 20.09 release, btw.